packages/site/cms-site/src/cookies/useLocalStorageCookieApi.ts

```diff
@@ -1,20 +1,31 @@
 import React from "react";
 import { useLocalStorage } from "usehooks-ts";
 
-import { CookieApiHook } from "./CookieApiHook";
+import { CookieApi } from "./CookieApi";
 
 const localStorageCookieApiKey = "comet-dev-cookie-api-consented-cookies";
 
+declare global {
+    interface Window {
+        cometLocalStorageCookieApi: ReturnType<CookieApi>;
+    }
+}
+
 /**
  * Only for use in development environment.
  */
-export const useDevCookieApi: CookieApiHook = (): ReturnType<CookieApiHook> => {
+export const useLocalStorageCookieApi: CookieApi = (): ReturnType<CookieApi> => {
```

Comment:
```suggestion
export const useLocalStorageCookieApi: CookieApi = (): CookieApi => {
```

Reply:
The return type was completely unnecessary here, as the whole function is already typed 😅